### PR TITLE
Cp2k pat

### PR DESCRIPTION
**Pull request by lucamar**
CP2K configuration file with perftools-cscs updated `builddependencies`

### CODE REVIEW

File: easybuild/easyconfigs/c/CP2K/CP2K-4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda.eb [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies = [
 ]
 
 builddependencies = [
-    ('perftools-cscs/%s' %patversion, EXTERNAL_MODULE),
+    ('perftools-cscs', '%s' %patversion, '', True),
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/CP2K/CP2K-4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda.eb]: mf = perftools-cscs/645-cuda
Should line 28 be `perftools-cscs/` instead ?

**Comment by jgphpc** [comment]
Can you show that 
/apps/daint/UES/jenkins/daint-gpu-jenkins-testPR/software/CP2K/4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda/bin/cp2k.psmp
generates a report ? If that's asking too much, let me know.

File: easybuild/easyconfigs/c/CP2K/CP2K-4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda.eb (hunk 1 above)
Comment (lucamar) [inline easybuild/easyconfigs/c/CP2K/CP2K-4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda.eb]: There's no need for the trailing `/`, EasyBuild will add it. Please check: 
```
/apps/daint/UES/6.0.UP02/craypat/easybuild/software/CP2K/4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda/easybuild/CP2K-4.1-CrayGNU-2016.11-cuda-8.0.54-pat-645-cuda-easybuild-devel
```

**Comment by lucamar** [comment]
Performance report files `.rpt` have already been created by `cp2k.psmp`: please check the path `/scratch/snx3000/lucamar/cp2k`

**Comment by jgphpc** [comment]
Thanks !

```
CrayPat/X:  Version 6.4.5 Revision 87dd5b8  01/23/17 15:37:24
Experiment:                   lite  lite/gpu
Number of PEs (MPI ranks):      16
Numbers of PEs per Node:         1  PE on each of  16  Nodes
Numbers of Threads per PE:   1,114
Number of Cores per Socket:     12
Execution start time:  Tue Mar 28 15:15:55 2017
System name and speed:  nid02294  2601 MHz (approx)
Intel haswell CPU  Family:  6  Model: 63  Stepping:  2

Avg Process Time:     2,100 secs
High Memory:       13,977.3 MBytes     873.6 MBytes per PE
I/O Read Rate:    67.110363 MBytes/sec
I/O Write Rate:   19.512511 MBytes/sec
```

**Pull request merged**